packages/lwc-language-server/src/lwc-server.ts

```diff
@@ -104,7 +104,7 @@ export default class Server {
             useDefaultDataProvider: false,
         });
 
-        this.context.configureProject();
+        await this.context.configureProject();
```

Comment:
Parts of the test suite were failing because this wasn't awaited. I believe this is the only functional change in the PR, and doesn't appear to be problematic.